src/db/tasks/abstract/RunOnConnection.ts

```diff
@@ -0,0 +1,24 @@
+import { connect } from '../..';
+import { Connection } from 'typeorm';
+
+export enum AvailableOperations {
+  RUN_MIGRATIONS = "runMigrations",
+  UNDO_LAST_MIGRATION = "undoLastMigration",
```

Comment:
Since we're calling the script `rollback`: 

```suggestion
  ROLLBACK = "undoLastMigration",
```